uktrade/data-hub-frontend
# Implement the paginated confirmed export wins table

## Description of change

Implements the paginated confirmed Export Wins page.

## Test instructions

1. With the application running against the sandbox (the API endpoint for Export Wins is not available yet)
2. Navigate to `/exportwins/confirmed`
3. You should see a paginated collection view of export wins under the "unconfirmed" tab
4. In your browser developer tools, set network throttling to a slower speed so you can better see the progress state
5. Click on one of the pagination links
6. There should appear a progress box on top of the current results
7. The page number should remain unchanged untill results for the requested page are successfully loaded
8. Once the requested page loads, the page number in the header and url param should update
9. If the page request rejects, the user should stay on the current page

## Screenshots

https://github.com/uktrade/data-hub-frontend/assets/2333157/66c792ee-3b3b-4d45-8b86-58e61fc95fcf

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
